### PR TITLE
Remove ExponentAtPriceOne From Gov Prop CreateConcentratedPool

### PR DESCRIPTION
**Pull request by stackman27**
Closes: #[5894](https://github.com/osmosis-labs/osmosis/issues/5894)

## What is the purpose of the change
Remove expAtone 

## Testing and Verifying
localosmosis tested 

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

**Review by czarcas7ic — APPROVED** [review]
Looks good after proto is fixed!

File: proto/osmosis/concentrated-liquidity/gov.proto [hunk 1]
```diff
@@ -51,12 +51,7 @@ message PoolRecord {
   string denom0 = 1 [ (gogoproto.moretags) = "yaml:\"denom0\"" ];
   string denom1 = 2 [ (gogoproto.moretags) = "yaml:\"denom1\"" ];
   uint64 tick_spacing = 3 [ (gogoproto.moretags) = "yaml:\"tick_spacing\"" ];
-  string exponent_at_price_one = 4 [
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.moretags) = "yaml:\"exponent_at_price_one\"",
-    (gogoproto.nullable) = false
-  ];
-  string spread_factor = 5 [
+  string spread_factor = 4 [
```
[the remark this answers is not in the document]
Comment (stackman27) [reply proto/osmosis/concentrated-liquidity/gov.proto:54]: i didn't know about this, is there documentation i can read? I always assumed that if we delete we just decrement the number

[the remark this answers is not in the document]
Comment (czarcas7ic) [reply proto/osmosis/concentrated-liquidity/gov.proto:54]: https://protobuf.dev/programming-guides/proto3/#deleting

[the remark this answers is not in the document]
Comment (czarcas7ic) [reply proto/osmosis/concentrated-liquidity/gov.proto:54]: So we should also reserve the field as well

[the remark this answers is not in the document]
Comment (stackman27) [reply proto/osmosis/concentrated-liquidity/gov.proto]: added https://github.com/osmosis-labs/osmosis/pull/5900/commits/a8ab0832e8cd6e93c70c6e45332260e5098e7f17

**Review by AlpinYukseloglu — APPROVED** [review]
LGTM!

**Pull request merged**